plugins/CoreHome/vue/src/index.ts

```diff
@@ -130,3 +135,14 @@ export { default as ContentIntro } from './ContentIntro/ContentIntro';
 export { default as ContentTable } from './ContentTable/ContentTable';
 export { default as AjaxForm } from './AjaxForm/AjaxForm.vue';
 export { default as Passthrough } from './Passthrough/Passthrough.vue';
+
+// for plugin modules loaded on demand that provide angularjs adapters, the adapters
+// have to be loaded through this event. this is because they must be defined before
+// the angular app is bootstrapped, so they must be in the javascripts/ folder.
+// but they must also be defined after CoreHome is loaded since that is where the
+// createAngularJsAdapter function is defined, and CoreHome is loaded after
+// javascripts/* scripts are. the event allows the adapters to be loaded right after
+// CoreHome is loaded at initial page load.
+//
+// Note: this will be removed in Matomo 5 and is only here for the interim.
+Matomo.postEvent('Matomo.addAngularJsAdapters', createAngularJsAdapter);
```

Comment:
Can this be removed now, as we directly merge this to Matomo 5?

Reply:
Yes, can definitely be removed :+1: